Gallery/TaylorDiagrams/NCL_taylor_3.py

```diff
@@ -0,0 +1,64 @@
+"""
+NCL_taylor_3.py
+===========
+This script illustrates the following concepts:
+   - Creating a basic Taylor diagram
+   - Adding labels to a Taylor diagram
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/taylor_3.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/taylor_3_lg.png
+"""
+
+###############################################################################
+# Import packages:
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+import geocat.viz as gv
+
+###############################################################################
+# Create dummy data:
+
+# Case A
+CA_ratio = [1.230, 0.988, 1.092, 1.172, 1.064, 0.966, 1.079, 0.781]
```

Comment:
Maybe add comments to these lines similar to the ones in Taylor_2 just in case a user only looks at this one and not the others